apps/web/src/actions/teams.ts

```diff
@@ -1,20 +1,26 @@
 "use server";
 
 // TODO: update team /api endpoints to be actions
-
 import { authenticatedAction } from "@/lib/safe-action";
-import { z } from "zod";
+import { boolean, string, z } from "zod";
 import { db } from "db";
 import { userHackerData, teams, invites } from "db/schema";
 import { eq } from "db/drizzle";
 import { revalidatePath } from "next/cache";
 import { getHacker } from "db/functions";
+import { returnValidationErrors } from "next-safe-action";
 
-export const leaveTeam = authenticatedAction(
-	z.null(),
-	async (_, { userId }) => {
+export const leaveTeam = authenticatedAction
+	.outputSchema(
```

Comment:
Not super familiar with the new way of the safe actions, but glanced at the docs and it seems like you use the `outputSchema` to define what the safe action will return and if not provided, it is inferred. Is this specific safe action not properly interpreted or is there another reason?

Reply:
Would make that a separate issue tbh

Reply:
Perhaps my question did not come across clearly. I am simply asking **why** this was used. Not whether it was an issue or something.

Reply:
Oh yeah I guess it's not strictly necessary

Reply:
Not a big deal. Was just curious what it was doing 👍